src/app/components/shell/notification-popover/AppShellNotificationPopover.vue

```diff
@@ -45,6 +45,9 @@ watch(
 				// Some default value before the real feed data gets populated
 				// on show.
 				itemsPerPage: 15,
+				extraData: {
```

Comment:
Not sure if you need this. We never load more or anything, right? It just fully reloads every time you open the popover?

I guess it's safer for future, though.